all-spark-notebook/Dockerfile

```diff
@@ -25,13 +25,19 @@ RUN conda install --quiet --yes \
     'r-irkernel=0.7*' \
     'r-ggplot2=2.2*' \
     'r-sparklyr=0.5*' \
-    'r-rcurl=1.95*' && conda clean -tipsy
+    'r-rcurl=1.95*' && \
+    conda clean -tipsy && \
+    fix-permissions $CONDA_DIR
```

Comment:
Setting the sticky bit gets us *almost* there, and I did set it. Good call. New files have the right owners, but not the right *permissions*. For that, I think we need [acls](https://unix.stackexchange.com/a/1315) or the ability to set umask for subsequent RUN commands, but AUFS doesn't support ACLs and I don't see a way to set umask, so I think we're stuck. I'd love to be wrong, though.